copulas/univariate/KDEUnivariate.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import numpy as np
+
+from copulas.univariate.UnivariateDistrib import UnivariateDistrib
+from scipy.stats import gaussian_kde
+import scipy.optimize as optimize
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class KDEUnivariate(UnivariateDistrib):
+    """ Use Kernel Density Estimation to infer univariate distribution """
+
+    def __init__(self):
+        super(KDEUnivariate, self).__init__()
```

Comment:
Please, add proper documentation. This include docstrings on a module and class level, also, on each method.

For methods, docstrings should contain at least:
- One line explanation of what the method does.
- Expected input parameters, with it's type and default values if any.
- Expected output format and type.
- Larger explanation of the side effects, exceptions raises, assumptions and/or restrictions to be called. In methods that implement mathematical procedures (pdf, cdf, ...) It could be a good idea to have the matematical formulation of what is being implemented in code.
- An small example of use (only on non-trivial cases)
For example:
```python
def add_nums(x,y):
    """Return the sum of two values.

    Args:
        :param x: First value to be sumed.
        :param y: Second value to be sumed.
        :type x: int or float
        :type y: int or float

    Returns:
        sum: int or float with the sum of the values.
    
    This function will sum the two values passed as arguments and return the result.
    It will raise TypeError if the arguments can not be summed.
    If one of the values if a float the result will also be a float.

   Example: 
        >>> add_nums(1, 1)
        2
    """
    return x + y
```
You can have a look at [PEP257](https://www.python.org/dev/peps/pep-0257/) if you need further indication on doing so.